WcaOnRails/lib/database_dumper.rb

```diff
@@ -547,6 +547,25 @@ def self.actions_to_column_sanitizers(columns_by_action)
       ),
     }.freeze,
     "locations" => :skip_all_rows,
+    "incidents" => {
+      where_clause: "",
+      column_sanitizers: actions_to_column_sanitizers(
+        copy: %w(id title public_summary digest_worthy resolved_at digest_sent_at created_at updated_at),
+        db_default: %w(private_description private_wrc_decision),
+      ),
+    }.freeze,
+    "incident_competitions" => {
+      where_clause: "",
+      column_sanitizers: actions_to_column_sanitizers(
+        copy: %w(id incident_id competition_id comments),
```

Comment:
Just to confirm: are these comments meant to be public information?

Reply:
actually I don't think so, I'll remove them from there!
I also just notice I don't display this information anywhere, so I'll fix this too.